src/views/lyrics.vue

```diff
@@ -641,6 +641,7 @@ export default {
     max-width: 460px;
     overflow-y: auto;
     transition: 0.5s;
+    scrollbar-width: none; //fire fox
```

Comment:
add the space between `//` and `fire`, and remove the space between `fire` and `fox`.